docs/core/entity/binary-sensor.md

```diff
@@ -13,8 +13,8 @@ Properties should always only return information from memory and not do I/O (lik
 
 | Name | Type | Default | Description
 | ---- | ---- | ------- | -----------
-| is_on| boolean | `None` | **Required**. If the binary sensor is currently on or off.
-| device_class | string | `None` | Type of binary sensor.
+| is_on| <code>bool &#124; None</code> | `None` | **Required**. If the binary sensor is currently on or off.
+| device_class | <code>str &#124; None</code> | `None` | Type of binary sensor.
```

Comment:
This should be `BinarySensorDeviceClass` type.

Reply:
```suggestion
| device_class | <code>SensorDeviceClass &#124; None</code> | `None` | Type of binary sensor.
```